mlengine/online-prediction/pom.xml

```diff
@@ -11,7 +11,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Can we intend this in alignment with other files?

Reply:
Fixed it!